src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -894,10 +926,41 @@ class IssueListOverview extends React.Component<Props, State> {
       location,
       router,
     } = this.props;
+    const links = parseLinkHeader(pageLinks);
     const query = this.getQuery();
     const queryPageInt = parseInt(location.query.page, 10);
-    const page = isNaN(queryPageInt) ? 0 : queryPageInt;
-    const pageCount = page * MAX_ITEMS + groupIds.length;
+    // Cursor must be present for the page number to be used
+    const page = isNaN(queryPageInt) || !location.query.cursor ? 0 : queryPageInt;
+    /**
+     * As items are removed from the current query, the total count might change.
+     * When you remove items from page 1 and get to the last page the totals will no longer
+     * match
+     */
+    const possibleCount = Math.max(page * MAX_ITEMS + groupIds.length, groupIds.length);
```

Comment:
I think this can be simplified to `page * MAX_ITEMS + groupIds.length` since it is always `>= groupIds.length`

Reply:
You're right!